js/components/image.js

```
@@ -225,34 +225,35 @@ Fliplet.FormBuilder.field('image', {
-          var scaledImage = loadImage.scale(img, options);
-          var imgBase64Url = scaledImage.toDataURL(mimeType, $vm.jpegQuality);
-          var flipletBase64Url = imgBase64Url + ';filename:' + file.name;
+            $vm.hasCorruptedImage = false;
+            $vm.isImageSizeExceeded = false;
+            $vm.value.push(file);
 
-          $vm.value.push(flipletBase64Url);
+            if (addThumbnail) {
+              var scaledImage = loadImage.scale(img, options);
+              var imgBase64Url = scaledImage.toDataURL(mimeType, $vm.jpegQuality);
+              var flipletBase64Url = imgBase64Url + ';filename:' + file.name;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)